napari/layers/_markers_layer/view/properties.py

```diff
@@ -31,8 +31,8 @@ def __init__(self, layer):
         sld.setValue(int(value))
         sld.valueChanged[int].connect(lambda value=sld: self.changeSize(value))
         self.sizeSlider = sld
-        self.grid_layout.addWidget(QLabel('size:'), 3, 0)
-        self.grid_layout.addWidget(sld, 3, 1)
+        self.grid_layout.addWidget(QLabel('size:'), 3, 0, 1, 2)
```

Comment:
See above — let's use kwargs in this situation to make the meaning more obvious here.

Reply:
as noted above I'd prefer to have a follow up PR that refactors all these numbers for all the properties widgets